src/benchmark/RemotePingPong.NET46/RemotePingPong.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+<PropertyGroup>
+  <Description>Akka.Remote x-plat performance benchmark</Description>
+  <Copyright>Copyright (c) Akka.NET Team</Copyright>
+  <AssemblyTitle>RemotePingPong</AssemblyTitle>
+  <AssemblyName>RemotePingPong</AssemblyName>
+  <VersionPrefix>1.3.0</VersionPrefix>
+  <Authors>Petabridge</Authors>
```

Comment:
Whoops, sorry about that - copied config from the `.csproj` for the NBench runner.